config/puma.rb

```diff
@@ -59,3 +59,6 @@
 
 # Allow puma to be restarted by `rails restart` command.
 plugin :tmp_restart
+
+# Run the Solid Queue supervisor inside of Puma for single-server deployments
+plugin :solid_queue if ENV["SOLID_QUEUE_IN_PUMA"]
```

Comment:
`SOLID_QUEUE_IN_PUMA` is used in solid_queue's docs. https://github.com/rails/solid_queue?tab=readme-ov-file#puma-plugin

Reply:
Thanks for finding a way to do this that will not break production.